notebooks/106-auto-device/106-auto-device.ipynb

```diff
@@ -0,0 +1,407 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "190e8e4c-461f-4521-ae7f-3491fa827ab7",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Automatic Device Selection\n",
+    "\n",
+    "The Auto Device (or AUTO in short) selects the most suitable device from the available compute devices by considering the network precision, power efficiency and processing capability. The network precision (if the network is quantized or not) is the first consideration to filter out the devices that cannot run the network efficiently.\n",
```

Comment:
```suggestion
    "The Auto Device (or AUTO in short) selects the most suitable device from the available compute devices by considering the network precision, power efficiency and processing capability. The network precision (if the network is quantized or not) is the first consideration to filter out the devices that cannot run the network efficiently.\n",
```
Please include a link (URL) where the people could find more information about Auto Device.